pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -389,6 +389,11 @@ type CloudConfig struct {
 		// Maybe if we're not running on AWS, e.g. bootstrap; for now it is not very useful
 		Zone string
 
+		// The AWS VPC flag enables the possibility to run the master components
+		// on a different aws account, on a different cloud provider or on-premise.
+		// If the flag is set also the KubernetesClusterTag must be provided
+		VPCID string
```

Comment:
Nit: I would probably name this VPC.  But whatever makes more sense to you is what we should use, and you're going to use it :-)